aarone_frontend/src/app/page_sections/banner/banner1/banner1.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="mt-bnr-inr overlay-wraper bg-parallax bg-top-center"  data-stellar-background-ratio="0.5"  style="background-image:url({{data.image}});">
```

Comment:
Is data an object you can configure? Wouldn't it be better to change it there so it would be easier to update later?